llvm/test/MC/RISCV/rv32zbkb-only-valid.s

```diff
@@ -14,3 +14,13 @@ zip t0, t1
 # CHECK-S-OBJ-NOALIAS: unzip t0, t1
 # CHECK-ASM: encoding: [0x93,0x52,0xf3,0x08]
 unzip t0, t1
+
+# Test the encoding used for zext.h for RV32.
+# CHECK-ASM: pack t0, t1, zero
```

Comment:
Note we get different printing depending on whether its printed from the assembler or disassembler. Assembler just echoes what we parsed. Disassembler will treat it as zext.h.